src/infrastructure/figma/transformers/figma-node-transformer.ts

```diff
@@ -0,0 +1,95 @@
+import {
+	buildDesignUrl,
+	buildInspectUrl,
+	buildLiveEmbedUrl,
+	getUpdateSequenceNumber,
+} from './utils';
+
+import type { AtlassianDesign } from '../../../domain/entities';
+import {
+	AtlassianDesignStatus,
+	AtlassianDesignType,
+	FigmaDesignIdentifier,
+} from '../../../domain/entities';
+import type { FileResponse, Node, NodeDevStatus } from '../figma-client';
+
+type TransformNodeToAtlassianDesignParams = {
+	readonly fileKey: string;
+	readonly nodeId: string;
+	readonly fileResponseWithNode: FileResponse;
+};
+
+export const transformNodeToAtlassianDesign = ({
```

Comment:
**note:** This code has been moved from `infrastructure/figma/figma-transformer.ts` and updated to work with `/v1/files/:fileKey?ids=:id` response.